packages/httpsnippet-client-api/test/__datasets__/issue-128/har.json

```diff
@@ -3,31 +3,17 @@
     "entries": [
       {
         "request": {
-          "allHeaders": {
-            "accept": "application/xml"
-          },
           "cookies": [],
-          "headers": [
-            {
-              "name": "Accept",
-              "value": "application/xml"
-            }
-          ],
+          "headers": [],
           "httpVersion": "HTTP/1.1",
           "method": "GET",
-          "postData": {
```

Comment:
This test is testing a GET request so it doesn't have any way of having a POST payload. Also removed the `Accept` headers from this because the OpenAPI definition for this one doesn't have anything about XML in it.